django-stubs/db/models/expressions.pyi

```diff
@@ -211,21 +221,30 @@ class Case(Expression):
         self, *cases: Any, default: Optional[Any] = ..., output_field: Optional[Field] = ..., **extra: Any
     ) -> None: ...
 
-class ExpressionWrapper(Expression):
-    def __init__(self, expression: Union[Q, Combinable], output_field: Field) -> None: ...
-
-class Col(Expression):
-    target: Field
-    alias: str
-    contains_column_references: Literal[True] = ...
-    possibly_multivalued: Literal[False] = ...
-    def __init__(self, alias: str, target: Field, output_field: Optional[Field] = ...) -> None: ...
+class Subquery(BaseExpression, Combinable):
+    template: str = ...
+    query: Query = ...
+    extra: Dict[Any, Any] = ...
```

Comment:
Since it comes from kwargs, keys are always strings

```suggestion
    extra: Dict[str, Any] = ...
```

Reply:
Well spotted, I was just moving this around but worth improving whilst we touch it.